internal/command/views/show_test.go

```diff
@@ -23,8 +25,14 @@ import (
 )
 
 func TestShowHuman(t *testing.T) {
+	redactedPath := "../../cloud/testdata/plan-json-basic/plan-redacted.json"
```

Comment:
I'm a little 🤔 about sharing the `cloud` package's test data with other packages this way - on the one hand, maintaining static JSON in test files is painful, but on the other hand, if I needed to change this `plan-redacted.json` file one day I wouldn't expect that my change might break a test in a completely different package. It's fine for now, but if we find more cross-over like this we should think about finding a new location for these shared files to make it clear that they're shared!